<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RectifiedImageViewWindow</class>
 <widget class="QDialog" name="RectifiedImageViewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="windowTitle">
      <string>toolBar</string>
     </property>
     <addaction name="actionBack"/>
     <addaction name="actionForward"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionBack">
   <property name="icon">
    <iconset theme="back">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset theme="forward">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
